frontend/javascripts/oxalis/geometries/plane.ts

```diff
@@ -4,20 +4,23 @@ import { getBaseVoxelFactors } from "oxalis/model/scaleinfo";
 import Dimensions from "oxalis/model/dimensions";
 import PlaneMaterialFactory from "oxalis/geometries/materials/plane_material_factory";
 import Store from "oxalis/store";
-import type { OrthoView, Vector3, Vector4 } from "oxalis/constants";
+import type { OrthoView, Vector3 } from "oxalis/constants";
 import constants, {
   OrthoViewColors,
   OrthoViewCrosshairColors,
   OrthoViewGrayCrosshairColor,
   OrthoViewValues,
 } from "oxalis/constants";
 
+export const PLANE_SUBDIVISION = 200;
```

Comment:
use in shader

Reply:
Maybe you could also add a comment on what the rationale behind this number is. It's probably not important that it's exactly 200, but explaining why it isn't 20 or 2000 might help future readers :)

Reply:
good point :) I just thought the value through (the initial value was more a gut feeling) and decreased it to 80 now (and added the explanation of course).